plugins/package-managers/gleam/src/main/kotlin/GleamPackageInfo.kt

```diff
@@ -311,6 +311,46 @@ internal data class DependencyPackageInfo(val depName: String, val dep: GleamTom
     }
 }
 
+/**
```

Comment:
I like moving related things next to each other, but on the other hand we sometimes group all constants together and like in previous change sort things alphabetically. I have a bit difficulties which one we follow. (I'm not against this change, but to me goals seem to slightly contradict)

Reply:
> I would still go for alphabetic ordering of global constants. And also if a function has multiple belonging constants, these should be sorted, IMO.

Sure, I agree with that. Basically, alphabetical ordering is ok as long as it does not contradict with cohesion / grouping of closely related things, right?